File: app/src/main/java/com/not/app/NeomuApp.java
package com.not.app;


import android.app.Application;

public class NeomuApp extends Application {
    private static NeomuApp mApp;
    @Override
    public void onCreate() {
        super.onCreate();

        mApp = this;
    }

    public static NeomuApp getApp() {
        return mApp;
    }
}

File: app/src/main/java/com/app/neomu_kotlin/app/NeoMuApp.kt
package com.app.neomu_kotlin.app

import android.app.Application

class NeoMuApp : Application() {

    override fun onCreate() {
        super.onCreate()
        mApp = this
    }

    companion object {
        lateinit var mApp: NeoMuApp

        fun getApp(): NeoMuApp {
            return mApp
        }
    }
}